copy.go

```diff
@@ -1,22 +1,28 @@
 package termenv
 
 import (
-	"github.com/aymanbagabas/go-osc52"
-)
+	"strings"
 
-func (o Output) osc52Output() *osc52.Output {
-	return osc52.NewOutput(o.tty, o.environ.Environ())
-}
+	"github.com/aymanbagabas/go-osc52/v2"
+)
 
 // Copy copies text to clipboard using OSC 52 escape sequence.
 func (o Output) Copy(str string) {
-	o.osc52Output().Copy(str)
+	s := osc52.New(str)
+	if strings.HasPrefix(o.environ.Getenv("TERM"), "screen") {
```

Comment:
Isn't that a logic that should be part of the osc52 package itself? But I guess it doesn't access the environment any longer?

Reply:
Yea, it doesn't access the environment any more.